apps/condo/domains/common/components/FormResetButton.tsx

```diff
@@ -1,19 +1,26 @@
-import React from 'react'
+import React, { useCallback } from 'react'
 import { Form } from 'antd'
+import { useRouter } from 'next/router'
 import { useIntl } from '@condo/next/intl'
 import { Button, CustomButtonProps } from './Button'
 
 export const FormResetButton: React.FC<CustomButtonProps> = (props) => {
     const intl = useIntl()
     const CancelMessage = intl.formatMessage({ id: 'Cancel' })
 
+    const { push, query: { id } } = useRouter()
+    const onCancel = useCallback((resetFields) => {
+        resetFields()
+        push(`/ticket/${id}`)
```

Comment:
it's a component from common domain and it's used in many other forms

Reply:
oh you are right